apps/etterlatte-beregning/src/main/kotlin/avkorting/AvkortingService.kt

```diff
@@ -86,6 +89,42 @@ class AvkortingService(
         }
     }
 
+    // Sjekke om Avkorting innvilga måneder overstyres på grunn av tidlig alderspensjon, og opprett oppgave om opphør
+    suspend fun opprettOppgaveHvisTidligAlderspensjon(
+        behandlingId: UUID,
+        brukerTokenInfo: BrukerTokenInfo,
+    ) {
+        val avkorting = avkortingRepository.hentAvkorting(behandlingId) ?: Avkorting()
```

Comment:
Her er det bedre å kaste feil hvis avkorting mangler for da er det noe som ikke stemmer.